open-hackathon-server/src/hackathon/hack/register_manager.py

```diff
@@ -141,6 +144,15 @@ def delete_registration(self, args):
                 self.db.delete_object(register)
                 hackathon = self.hackathon_manager.get_hackathon_by_id(register.hackathon_id)
                 self.__update_register_stat(hackathon)
+
+                user = self.user_manager.get_user_by_id(register.user_id)
```

Comment:
You can get user by `user = register.user` and skip next two lines since `register.user` cannot be None because of the Foreign Key constraint.